src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -164,6 +179,27 @@ private static void GetTempPath(ref ValueStringBuilder builder)
                 throw Win32Marshal.GetExceptionForLastWin32Error();
 
             builder.Length = (int)result;
+
+            static uint GetTempPathW(int bufferLen, ref char buffer)
+            {
+                delegate* unmanaged<int, char*, uint> func = s_GetTempPathWFunc;
+                if (func == null)
+                {
+                    func = s_GetTempPathWFunc = GetGetTempPathWFunc();
+                }
+
+                int lastError;
+                uint retVal;
+                fixed (char* ptr = &buffer)
+                {
+                    Marshal.SetLastSystemError(0);
+                    retVal = func(bufferLen, ptr);
+                    lastError = Marshal.GetLastSystemError();
+
+                }
+                Marshal.SetLastPInvokeError(lastError);
```

Comment:
the newline suggestion was for _after_ the brace.
```suggestion
                    lastError = Marshal.GetLastSystemError();
                }
                
                Marshal.SetLastPInvokeError(lastError);
```